moonstreamdb-v3/moonstreamdbv3/models_indexes.py

```diff
@@ -153,6 +153,59 @@ class EthereumReorgs(EvmBasedReorgs):
     __tablename__ = "ethereum_reorgs"
 
 
+class SepoliaBlockIndex(EvmBasedBlocks):
+    __tablename__ = "ethereum_sepolia_blocks"
+
+
+class SepoliaTransactionIndex(EvmBasedTransactions):
+    __tablename__ = "ethereum_sepolia_transactions"
+
+    block_number = Column(
+        BigInteger,
+        ForeignKey("ethereum_sepolia_blocks.block_number", ondelete="CASCADE"),
+        nullable=False,
+        index=True,
+    )
+
+
+class SepoliaLogIndex(EvmBasedLogs):
+
+    __tablename__ = "ethereum_sepolia_logs"
```

Comment:
Lets call it 
`EthereumSepoliaLogIndex` with `__tablename__ = "ethereum_sepolia_logs"` 
or 
`SepoliaLogIndex` with `__tablename__ = "sepolia_logs"`